### PR TITLE
Fix eventmachine dependency for both Windows and other platforms

### PR DESCRIPTION
**Pull request by southwolf**
Fix #896 and #887 for good. Both sides should be happy :smile:

### CODE REVIEW

**Comment by southwolf** [comment]
Seems that Travis CI fails randomly. It works now.

**Comment by kreynolds** [comment]
+1

File: sensu.gemspec [hunk 1]
```diff
@@ -16,7 +16,11 @@ Gem::Specification.new do |s|
   s.add_dependency "json" if RUBY_VERSION < "1.9"
   s.add_dependency "multi_json", "1.11.0"
   s.add_dependency "uuidtools", "2.1.4"
-  s.add_dependency "eventmachine", "1.0.3"
+  if RUBY_PLATFORM =~ /mingw/
```
Comment (portertech) [inline sensu.gemspec]: This cannot work when publishing the gem to rubygems, as it will use the platform it is pushed from.

**Comment by portertech** [comment]
The Ruby platform is evaluated when the gem is built and published to Rubygems, not on install etc :( Going to close this in favour of https://github.com/sensu/sensu/pull/914 which may be replaced with a https://github.com/sensu/sensu-build change to install the latest eventmachine using devkit.

**Comment by sodabrew** [comment]
The latest eventmachine definitely works on Windows; if you're seeing any trouble please report it?

**Pull request closed without merging**